data/txHandler.go

```diff
@@ -90,6 +91,7 @@ func MakeTxHandler(txPool *pools.TransactionPool, ledger *Ledger, net network.Go
 		backlogQueue:          make(chan *txBacklogMsg, txBacklogSize),
 		postVerificationQueue: make(chan *txBacklogMsg, txBacklogSize),
 		net:                   net,
+		txidCache:             makeTxidCacheSyncMap(txBacklogSize),
```

Comment:
Is there value in making this a bit larger? Maybe 2x? There is an implicit 2x of the `txBacklogSize` in the cache implementation with `cur` and `prev` but we could be getting unique transactions from at least 4 peers and at current txBacklogSize this is only 2.5Mb of memory consumed (256 bytes (hash size) * 5000 (backlog size) * 2) for potentially a lot of work saved down the line

Reply:
yes, I planned do 2x to have at least 2 full blocks of data in worse case, and currently selecting between `container.Ring` and 2x